cubed/array_api/creation_functions.py

```diff
@@ -90,7 +95,7 @@ def empty_virtual_array(
     shape, *, dtype=None, device=None, chunks="auto", spec=None, hidden=True
 ) -> "Array":
     if dtype is None:
-        dtype = nxp.float64
+        dtype = f'f{p}'
```

Comment:
Rather than use strings here, I would change cubed/backend_array_api.py to expose default float and int dtypes that can be referred to directly from here and other places like it.

Eventually we can use the [`default_dtypes`](https://data-apis.org/array-api/2023.12/API_specification/generated/array_api.info.default_dtypes.html#array_api.info.default_dtypes) inpection API from the array API, but that is only available from 2023.12. Still, I think it's worth doing something similar in the backend even if the implementation is currently hardcoded for JAX.